src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -26,6 +26,15 @@ or `pip install --pre toil-scripts` for cutting edge development version.
 
 Type: `toil-exome` to get basic help menu and instructions
 
+Our software lead prefers installation of python projects into a virtualenv. 
```

Comment:
This was in *no way* meant to be a dig, merely an explanation of why I'm asking the user to install it into a virtualenv, something a user may never have seen as part of pip installing a package.  I'm not disagreeing with your point whatsoever. I'll go ahead and substitute it with your more substantiative explanation.